src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -15,9 +14,9 @@ public class MovieSession {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
-    @ManyToOne(fetch = FetchType.LAZY)
+    @ManyToOne
```

Comment:
You should not change this relationship, return them
```suggestion
    @ManyToOne(fetch = FetchType.LAZY)
```